src/devices/missil_ml0757.c

```diff
@@ -105,7 +105,8 @@ static int missil_ml0757_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     if (flag_rwp) { // Rainwall and wind
         /* clang-format off */
         data = data_make(
-                "model",            "",             DATA_STRING, _X("ML0757-RainWind","Missil 0757 Rain/Wind"),
+                "model",            "",             DATA_STRING, "Missil-ML0757",
+                "subtype",          "Sensor(s)",    DATA_STRING, "RainfallWind",
```

Comment:
`subtype` is for e.g. security devices like PIR vs door contact where a single unit is of a specific function. It's ok to not have a subtype here, some messages will have the rain value, some the temp, but overall you get both. In other words subtype should be static for each device.